package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.2.0",
   "author": "John Foster",
   "description": "Support for using highlight.js to syntax highlight cURL commands.",
-  "main": "src/language/curl.js",
+  "main": "dist/curl.min.js",
```

Comment:
This isn't right.  This should point to the original module for `npm`, etc...  `dist` is something that someone would manually choose to load in their browser.

Reply:
ie, it was correct as-is

Reply:
thanks, i didn't know that